test/statetest/statetest_loader.cpp

```diff
@@ -341,6 +382,12 @@ static void from_json_tx_common(const json::json& j, state::Transaction& o)
         for (const auto& hash : *it)
             o.blob_hashes.push_back(from_json<bytes32>(hash));
     }
+    else if (const auto au_it = j.find("authorizationList"); au_it != j.end())
+    {
+        o.authorization_list = from_json<state::AuthorizationList>(*au_it);
+        if (o.type <= state::Transaction::Type::eip1559)
```

Comment:
Why only update the transaction type conditionally?

Also I noticed we don't read the transaction type directly from JSON but EEST may produce these.

Reply:
I'm not sure why, I think, @pdobacz, you wrote this, do you remember anything?

Reply:
Removed the condition when rebasing.

Reply:
I don't remember sorry, and I cannot see the full diff here, I don't know the context (what followed after the `if`) to aid my memory. If I could see my original commit maybe, but it seems I don't have that either.